src/components/NavigationBar.tsx

```diff
@@ -0,0 +1,22 @@
+import { H2 } from "./Text";
+
+import BackIcon from "@/assets/back-icon.svg";
+import MenuIcon from "@/assets/menu-icon.svg";
+
+interface Props {
+  children: string;
+}
+
+const NavigationBar = ({ children }: Props) => {
+  return (
+    <div className="flex items-center bg-tertiary-700 w-full px-5 py-3">
+      <BackIcon className="mr-5" />
+      <H2 className="text-neutral-0 w-full">{children}</H2>
+      <div className="flex flex-end">
+        <MenuIcon className="w-full " />
+      </div>
```

Comment:
```suggestion
      <BackIcon className="mr-5" />
      <H2 className="text-neutral-0 grow">{children}</H2>
      <MenuIcon />
```
이렇게 작성해도 좋을 것 같아요!
근데 스펙에서 네비게이션 우측에 들어가는 아이콘이 무조건 메뉴아이콘 하나인가요??